.circleci/config.yml

```diff
@@ -90,52 +111,58 @@ jobs:
                 name: Integration Tests - Usage Apps Rendering
                 command: |
                     . venv/bin/activate
-                    pytest --nopercyfinalize --headless tests/test_usage.py
+                    python -m pytest --nopercyfinalize --headless tests/test_usage.py
                 when: always
 
             - run:
                 name: Integration Tests - Interactions
                 command: |
                     . venv/bin/activate
-                    pytest --headless tests/test_interactions.py
+                    python -m pytest --headless tests/test_interactions.py
                 when: always
 
             - run:
                 name: Integration Tests - Callbacks
                 command: |
                     . venv/bin/activate
-                    pytest --headless tests/test_callbacks.py
+                    python -m pytest --headless tests/test_callbacks.py
                 when: always
 
             - run:
                 name: Capture Percy Snapshots
                 command: |
                     . venv/bin/activate
-                    pytest --headless tests/test_percy_snapshot.py
+                    python -m pytest --headless tests/test_percy_snapshot.py
                 when: always
 
-    "python-3.7":
+    python-3-6:
         <<: *test-template
-        docker:
-            - image: circleci/python:3.7-stretch-browsers
-              auth:
-                  username: dashautomation
-                  password: $DASH_PAT_DOCKERHUB
-                  
+        executor: python-3-6-executor
+
         environment:
             PERCY_ENABLE: False
 
-
-workflows:
-    version: 2
+workflows:    
+    version: 2.1
     build:
         jobs:
-            - "python-3.6":
+            - python-3-11:
                 context: dash-docker-hub
```

Comment:
not new in this PR, but isn't this `context` redundant with the `auth` you already have defined in each executor?